SECURITY.md

```diff
@@ -0,0 +1,5 @@
+If you have discovered a security vulnerability, we appreciate your help by disclosing it to us in a responsible manner.
+Please refer to https://opentitan.org/cvd-policy for a description of our disclosure process.
```

Comment:
Link is dead but I assume it will be published soon?

Reply:
Yes, my plan is to update the website as soon as this is merged.